MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/Media/FoldersTreeView.js

```diff
@@ -0,0 +1,52 @@
+import AbstractTreeView from '../Tree/AbstractTreeView'
+import Folder           from '../../Model/Folder/Folder'
+import ApplicationError from '../../../Service/Error/ApplicationError'
+import Application      from '../../Application'
+
+/**
+ * @class FoldersTreeView
+ */
+class FoldersTreeView extends AbstractTreeView
+{
+    /**
+     * Initialize
+     * @param {FoldersTree} foldersTree
+     * @param {string}      language
+     */
+    initialize({foldersTree, language}) {
+        this._foldersTree = foldersTree;
+        this._language = language
+    }
+
+    /**
+     * Get the tree template
+     * @return {Object}
+     * @private
+     */
+    _getTreeTemplate() {
+        return this._renderTemplate('Folder/foldersTreeView',
+            {
+                foldersTree : this._foldersTree.models,
+                language: this._language
+            }
+        );
+    }
+
+    /**
+     * @param {Object} event
+     * @param {Object} ui
+     * @private
+     */
+    _sortAction(event, ui) {
```

Comment:
By using sortable as in node context, the user will expect to get orderable folders and ui will render as if.
It should be more natural to use droppable.